src/OSPSuite.Core/Domain/Data/QuantityInfo.cs

```diff
@@ -26,19 +21,17 @@ public class QuantityInfo
       public int OrderIndex { get; set; }
 
       [Obsolete("For serialization")]
-      public QuantityInfo() : this(string.Empty, new List<string>(), QuantityType.Undefined)
+      public QuantityInfo() : this(new List<string>(), QuantityType.Undefined)
       {
       }
 
       /// <summary>
       ///   Create a new quantity info
       /// </summary>
-      /// <param name = "name">Name of quantity but could be set to sthg else. Only used in DataColumn.QuantityName</param>
       /// <param name = "quantityPath">Full path of quantity</param>
       /// <param name = "quantityType">Type of quantity (such as species, observer, parameter). </param>
-      public QuantityInfo(string name, IEnumerable<string> quantityPath, QuantityType quantityType)
```

Comment:
Remote the name of the quantity info which is ALWAYS set to the name of the DataColumn except in our tests. Since it can only be created programmatically, remove